src/platform/utilities/api/index.js

```diff
@@ -1,8 +1,68 @@
 import * as Sentry from '@sentry/browser';
 
 import environment from '../environment';
+import ENVIRONMENTS from '../../../site/constants/environments';
 import localStorage from '../storage/localStorage';
 
+let sessionTimeoutMinutesLengthSSO = 0;
+const SSOeEndpoint = () => {
+  let environmentPrefix;
+  switch (ENVIRONMENTS.BUILDTYPE) {
+    case ENVIRONMENTS.VAGOVSTAGING:
+      environmentPrefix = 'sqa.';
+      break;
+    case ENVIRONMENTS.VAGOVDEV:
+      environmentPrefix = 'int.';
+      break;
+    default:
+      environmentPrefix = '';
+  }
+
+  return `https://${environmentPrefix}eauth.va.gov/keepalive`;
+};
+
+function checkAndUpdateSSOeSession() {
+  const sessionExpiration = localStorage.getItem('sessionExpirationSSO');
+
+  if (sessionExpiration) {
+    const minutesUntilExpiration =
+      (sessionExpiration.getTime() - Date.now()) / (1000 * 60);
+
+    if (minutesUntilExpiration / sessionTimeoutMinutesLengthSSO < 0.67) {
+      // We want to minimize external keepalive calls, so this functions
+      // as self-enforced rate limiting
+      return;
+    }
```

Comment:
At a glance this looks like it would be dividing by 0.  Also it isn't really clear what the 0.67 threshold is supposed to be.

Reply:
my thought was if an SSO session isn't active, then technically it would be 0. it could just as easily be `null`, since this function should only ever get called if that value is non-zero.

does something like this before the `if` block more accurately express the intent of that check?
```js
const percentageOfTimeoutThreshold = .67
```